source/features/easy-toggle-commit-messages.tsx

```diff
@@ -0,0 +1,29 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function toggleCommitMessage(event: MouseEvent): void {
+	const elementClicked = event.target as HTMLElement;
+	const commitBox = elementClicked.closest('.js-commits-list-item')!;
```

Comment:
This is just recreating `delegate`, so we should use that.

Reply:
I've changed `easy-toggle-files` to use `delegate` too, since that's where I took the base code from.